distribution/std-lib/Base/src/Data/Number/Extensions.enso

```diff
@@ -106,3 +106,10 @@ Decimal.parse text =
 Number.compare_to : Number -> Ordering
 Number.compare_to that = if this == that then Ordering.Equal else
     if this > that then Ordering.Greater else Ordering.Less
+
+## Right bit shift on small integers.
```

Comment:
Please document the `zero_fill` argument, I did not understanding until I read the code.